packages/stark-core/src/common/error/error.ts

```diff
@@ -1,8 +1,11 @@
+/* tslint:disable completed-docs*/
```

Comment:
The correct format is: `/* tslint:disable:completed-docs */` otherwise all rules are disabled ;)